assets/js/modules/analytics-4/utils/report-validation.test.js

```diff
@@ -157,4 +158,135 @@ describe( 'Analytics 4 Reporting API validation', () => {
 			).toBe( false );
 		} );
 	} );
+
+	describe( 'isValidOrders', () => {
+		it( 'should return TRUE if an array of valid order objects is passed', () => {
+			expect(
+				isValidOrders( [
+					{
+						metric: {
+							metricName: 'totalUsers',
+						},
+						desc: false,
+					},
+					{
+						metric: {
+							metricName: 'sessions',
+						},
+						desc: true,
+					},
+					{
+						dimension: {
+							dimensionName: 'date',
+						},
+						desc: true,
+					},
+					{
+						dimension: {
+							dimensionName: 'sessionDefaultChannelGrouping',
+						},
+					},
+				] )
+			).toBeTruthy();
+		} );
+
+		it( 'should return FALSE if an array is not passed', () => {
+			expect( isValidOrders( null ) ).toBeFalsy();
+			expect( isValidOrders( 'test' ) ).toBeFalsy();
+			expect( isValidOrders( { test: 123 } ) ).toBeFalsy();
```

Comment:
Same here, we need to use `.toBe( false )` instead of `.toBeFalsy()`.

Reply:
Thanks, and as above.